lib/presentation/workout_actions/workout_action_route.dart

```diff
@@ -24,6 +24,13 @@ class WorkoutActionPage extends StatelessWidget {
     final editWorkoutBloc = BlocProvider.of<WorkoutActionBloc>(context);
     return Scaffold(
       body: _renderWorkouts(editWorkoutBloc, editWorkoutState),
+      floatingActionButton: FloatingActionButton(
+        onPressed: () => {
+          editWorkoutBloc.add(new CreateProgramEvent(programName: "Test", description: "Test description", duration: 20))
```

Comment:
Maybe we can move this out of `workout_action` to the home/discover route because this page can only be accessed if there is an existing workout.